src/runtime/components/Table.vue

```diff
@@ -206,12 +206,23 @@ const columns = computed<TableColumn<T>[]>(() => props.columns ?? Object.keys(da
 const meta = computed(() => props.meta ?? {})
 
 const ui = computed(() => tv({ extend: tv(theme), ...(appConfig.ui?.table || {}) })({
-  sticky: props.sticky,
+  sticky: props.sticky === true || props.sticky === 'header',
+  stickyFooter: props.sticky === true || props.sticky === 'footer',
   loading: props.loading,
   loadingColor: props.loadingColor,
   loadingAnimation: props.loadingAnimation
 }))
 
+const hasFooter = computed(() => {
```

Comment:
Would you mind explaining the logic of this? Wouldn't it be simpler with `columns.value.map`? 🤔